README.md

```diff
@@ -202,6 +202,50 @@ Defaults to `true`.
 **Required**.
 The version of terraform to install and use (e.g. `1.2.1`).
 
+## Terraform Format
+
+This is used to ensure formatting on PRs. It is not included in the standard workflow.
+It will fail if `terraform fmt` has any changes and create a PR with the changes to fix formatting with your PR as the target.
+The formatting is recursive, so you can use one action for the entire repo.
+
+Sample yaml file:
+```yaml
+# .github/workflows/terraform-format.yml
+name: terraform-format
+
+on:
+  pull_request:
+    paths:
+      - 'terraform/**'
+      - '.github/workflows/terraform-format.yml'
+jobs:
+  format:
+    runs-on: [self-hosted, Linux, AWS]
+    timeout-minutes: 5
+    steps:
+      - name: checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Format
+        uses: Brightspace/terraform-workflows/actions/format@v3
+        with:
+          terraform_version: 0.14.4
+          workspace_path: 'terraform'
```

Comment:
Workspace has meaning within terraform parlance of basically, "where you run init". This doesn't like up with that so I think it could use a different name.

Perhaps base path?